src/include/distributed/shard_split.h

```diff
@@ -28,6 +29,23 @@ typedef enum SplitOperation
 	ISOLATE_TENANT_TO_NEW_SHARD
 } SplitOperation;
 
+/*
+ * In-memory mapping of a split child shard.
+ */
+typedef struct ShardSplitInfo
```

Comment:
This is more suitable to be defined in shardsplit_shared_memory.h right?

Reply:
Done.